src/components/TimeZoneCard/TimeZoneCard.js

```diff
@@ -30,10 +27,8 @@ export function TimeZoneCard({
                         <h4>{ country }</h4>
                     </div>
                     <LocationTime
-                        time={ time }
-                        day={ day }
-                        date={ date }
                         alignment={ "right" }
+                        timezone={ "Australia/Sydney" }
```

Comment:
add conditoinally

Reply:
Will update after adding the timezone api to fetch timezone.